tgui/packages/tgui/interfaces/CameraConsole.js

```diff
@@ -0,0 +1,138 @@
+import { filter, sortBy } from 'common/collections';
+import { flow } from 'common/fp';
+import { classes } from 'common/react';
+import { createSearch } from 'common/string';
+import { Fragment } from 'inferno';
+import { useBackend, useLocalState } from '../backend';
+import { Button, ByondUi, Input, Section } from '../components';
+import { refocusLayout, Window } from '../layouts';
+
+/**
+ * Returns previous and next camera names relative to the currently
+ * active camera.
+ */
+const prevNextCamera = (cameras, activeCamera) => {
+  if (!activeCamera) {
+    return [];
+  }
+  const index = cameras.findIndex(camera => (
+    camera.name === activeCamera.name
+  ));
+  return [
+    cameras[index - 1]?.name,
+    cameras[index + 1]?.name,
+  ];
+};
+
+/**
+ * Camera selector.
+ *
+ * Filters cameras, applies search terms and sorts the alphabetically.
+ */
+const selectCameras = (cameras, searchText = '') => {
+  const testSearch = createSearch(searchText, camera => camera.name);
+  return flow([
+    // Null camera filter
+    filter(camera => camera?.name),
+    // Optional search term
+    searchText && filter(testSearch),
+    // Slightly expensive, but way better than sorting in BYOND
+    sortBy(camera => camera.name),
+  ])(cameras);
+};
+
+export const CameraConsole = (props, context) => {
+  const { act, data, config } = useBackend(context);
+  const { mapRef, activeCamera } = data;
+  const cameras = selectCameras(data.cameras);
+  const [
+    prevCameraName,
+    nextCameraName,
+  ] = prevNextCamera(cameras, activeCamera);
+  return (
+    <Window
+      width={870}
+      height={708}
+      resizable>
+      <div className="CameraConsole__left">
+        <Window.Content scrollable>
+          <CameraConsoleContent />
+        </Window.Content>
+      </div>
+      <div className="CameraConsole__right">
+        <div className="CameraConsole__toolbar">
+          <b>Camera: </b>
```

Comment:
Prefer semantic elements rather than elements-that-are-just-spans-with-styles (i.e. span with a semantic class that resolves to being bold over just using `b`).

Not requiring for this PR, something to hit early so others can take advantage of them, though.

Reply:
There is no need to, we are not SEO optimizing our UIs.

Reply:
Not from an SEO point of view, but from a "keep your UI elements looking the same" point of view. `<b>` doesn't tell me what this is - is it a label, is it a title, is it just something we want to say loudly?

Semantic class naming isn't just for SEO, it's useful for code maintenance and style refactoring on a larger level.